contrib/mac/framework/delete-all-rpaths.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+if [ $# -lt 1 ]; then
+	echo "usage: $(basename $0) lib1 [lib2 ...]" 2>&1
```

Comment:
use spaces for shell scripts - either 2 or 4 as long as you're consistent within the file